lib/builder.js

```diff
@@ -493,7 +496,10 @@ Builder.prototype.transpileTree = function(tree) {
     modules: 'amdStrict',
     exportModuleMetadata: true,
     moduleIds: true,
-    sourceMaps: true
+    sourceMaps: true,
+    resolveModuleSource: function(moduleSource, fileName) {
```

Comment:
I don't think you need the extra callback, you can likely just pass `resolveModuleSource: amdNameResolver`